dotcom-rendering/src/components/Masthead/HighlightsCard.stories.tsx

```diff
@@ -105,3 +105,19 @@ export const WithStarRating: Story = {
 	parameters: {},
 	name: 'With Star Rating',
 };
+
+export const WithPodcastSeriesImage: Story = {
```

Comment:
The podcast image is coming up correctly, nice! 

The story is also showing a gallery pill rather than a podcast one, which just feels a bit weird, but also it's not that important considering the point is to showcase the series image in the story rather than the whole design